frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -394,6 +394,31 @@ describe("scenarios > question > notebook", () => {
     cy.findByText("ID").should("not.exist");
   });
 
+  it("should treat max/min on a name as a string filter (metabase#21973)", () => {
+    startNewQuestion();
+    cy.contains("Sample Database").click();
+    cy.contains("People").click();
+    cy.contains("Pick the metric").click();
+    popover().within(() => {
+      cy.findByText("Maximum of ...").click();
+      cy.findByText("Name").click();
+    });
+    cy.findByText("Pick a column to group by").click();
+    popover().within(() => {
+      cy.findByText("Source").click();
+    });
+
+    cy.get(".Icon-filter").click();
+    popover().within(() => {
+      cy.findByText("Max of Name").click();
+    });
+
+    cy.findByText("Is").click();
+    cy.findByText("Starts with").click();
+
+    cy.findByText("Case sensitive").click();
```

Comment:
The whole test can be simplified and made faster two times by using the API to set things up.
```js
const questionDetails = {
  name: "21973",
  query: {
    "source-table": PEOPLE_ID,
    aggregation: [["max", ["field", PEOPLE.NAME, null]]],
    breakout: [["field", PEOPLE.SOURCE, null]],
  },
  display: "table",
};

cy.createQuestion(questionDetails, { visitQuestion: true });

cy.findByText("Filter").click();
cy.findByTestId("sidebar-right").within(() => {
  cy.findByText("Max of Name").click();

  cy.findByText("Is").click();
});

cy.findByText("Starts with").click();

cy.findByText("Case sensitive").click();
```

Reply:
Excellent suggestion, thanks. I've adopted that improvement.